keadm/cmd/keadm/app/cmd/debug/collect.go

```diff
@@ -0,0 +1,225 @@
+package debug
+
+import (
+	"flag"
+	"fmt"
+	"github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/common"
+	types "github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/common"
+	"github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/util"
+	"github.com/spf13/cobra"
```

Comment:
split import into groups.

Reply:
The project package and the outer package split with  blank line？

Reply:
> split import into groups.

The project package and the outer package split with blank line？

Reply:
yep